Formula/gdu.rb

```diff
@@ -18,8 +18,6 @@ class Gdu < Formula
   conflicts_with "coreutils", because: "both install `gdu` binaries"
 
   def install
-    ENV["TZ"] = "UTC"
```

Comment:
Took the opportunity to remove this line, wasn't necessary since the `time` method takes care of UTC.